.github/workflows/ci.yml

```diff
@@ -582,21 +582,95 @@ jobs:
             bash -euo pipefail -c \
             "./build_tools/cmake/build_riscv.sh && tests/riscv32/smoke.sh"
 
-  riscv64:
-    needs: [should_run, build_all, build_tf_integrations]
+  build_riscv64:
```

Comment:
This job builds with assertions, which we want for testing, but I think for benchmarking you're going to want no assertions

Reply:
Good point. Then I guess we can't share the test and benchmark builds. Splitted them into two standalone builds.